mavlink-protocol/src/main/java/io/dronefleet/mavlink/protocol/MavlinkFrameReader.java

```diff
@@ -78,5 +84,13 @@ public void drop() throws IOException {
         in.rollback();
         in.skip(1);
         in.commit();
+        bytesDropped++;
+    }
+
+    /**
+     * @return the amount of bytes dropped.
+     */
+    public int getNumBytesDropped() {
```

Comment:
I prefer just `getBytesDropped` (the return type makes it obvious that it's a number), or `getBytesDroppedCount` (less preferable). Up to you though.